tests/e2e/__testers__.ts

```diff
@@ -35,32 +74,26 @@ export const testTemplate = (params: {
             },
           },
         }) as ClientBase
-      const getPrismaAdmin = async () =>
-        new (await getPrismaClientModule()).PrismaClient({
-          datasources: {
-            db: {
-              url: `${databaseUrlBase}/postgres`,
-            },
-          },
-        }) as ClientBase
+
+      const getPrismaAdmin = async () => {
+        return params.getPrismaAdmin(databaseUrlBase, (await getPrismaClientModule()).PrismaClient)
+      }
 
       const dropTestDatabase = async () => {
-        const prisma = await getPrismaAdmin()
-        try {
-          await prisma.$executeRawUnsafe(`DROP DATABASE ${databaseName} WITH (FORCE);`)
-        } catch (error) {
-          const isDatabaseNotFoundErorr = error instanceof Error && error.message.match(/does not exist/)
-          if (!isDatabaseNotFoundErorr) throw error
-        } finally {
-          await prisma.$disconnect()
-        }
+        await ctx.run(`prisma migrate reset --force`, { reject: true })
+        return dropDatabase(await getPrismaAdmin(), databaseName, datasourceProvider)
+      }
+
+      const initTestDatabase = async () => {
```

Comment:
```suggestion
      const createTestDatabase = async () => {
```

It was a bit hard for me to follow/remember what `init` means. This function's job is very narrow, I think this better captures its purpose.